src/deployment/post-deployment/tasks/run-console-app.yaml

```diff
@@ -1,18 +1,22 @@
-﻿# Runs a console app on an Azure Web Application in the specified location
-# ref: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/start-process?view=powershell-7#example-7--specifying-arguments-to-the-process
+﻿# Task to run a console app with args
 parameters:
```

Comment:
Would this only ever work on windows? Should we make that explicit?

Reply:
I believe this would also work on linux, the `Powershell@2` task will run PS Core on Linux